users/klackygears/rules.mk

```diff
@@ -0,0 +1,12 @@
+SRC += klackygears.c
+
+#EXTRAFLAGS += -flto  compiles, does weird things to the helix
+
+
+ifeq ($(strip $(TAP_DANCE_ENABLE)), yes)
+  SRC += tap_dances.c
+endif
+
+ifeq ($(strip $(MACROS_ENABLED)), yes)
+    OPT_DEFS += -DMACROS_ENABLED
+endif
```

Comment:
I might have missed it, but I don't see this used anywhere.

Reply:
It's not used in the brain keymap but, others should I remove it for this pr?